functions/src/apis/bikes.ts

```diff
@@ -0,0 +1,161 @@
+import {Response as responseType} from "express";
+import {v4 as uuidv4} from "uuid";
+import {db} from "../admin";
+
+// define types
+type location = {
+  latitude: number,
+  longitude: number,
+}
+
+type entryType = {
+    bike_id: string,
+    name: string,
+    owner: string,
+    photo: string,
+    release: boolean,
+    num_ratings: number,
```

Comment:
I don't see num_ratings in the ERD Jon put together. Let us know what this is in Slack so we know what to do with it moving forward.

Reply:
Starting a new channel in Slack for database-related things including this and another issue...

Reply:
Great thanks! I added the topic to the #database channel on slack.

Note, for the next pull-request, I will add the field **description**